qemu/tests/cfg/win_virtio_driver_update_test.cfg

```diff
@@ -18,6 +18,10 @@
     sub_test = win_virtio_driver_update_test
     devcon_dirname = "win7_"
     vio_driver_chk_cmd = 'driverquery /si | find /i "%s"'
+    uninst_store_cmd = "pnputil /delete-driver %s /uninstall /force"
```

Comment:
I suppose this param definition on all cfg files could be removed as it has the pre-definition in the script, let's take the benefit.

Reply:
Updated. 
Thank you.